src/react/DevicesPage.jsx

```diff
@@ -187,21 +212,24 @@ export default ({ astarte, history }) => {
   );
 };
 
-const TablePagination = ({ active, max, onPageChange }) => {
+const TablePagination = ({ active, canLoadMorePages, max, onPageChange }) => {
```

Comment:
Maybe it would be clearer to explicit `page` in all the props:
`active` -> `activePage`
`max` -> `maxPage`
Right now `active` makes me think about whether TablePagination should be visible or not